packages/autocomplete-core/src/onKeyDown.ts

```diff
@@ -99,6 +99,10 @@ export function onKeyDown<TItem extends BaseItem>({
     event.preventDefault();
 
     store.dispatch(event.key, null);
+
```

Comment:
```suggestion

    // The Escape key signals the end of a user interaction with the autocomplete.
    // At this point, we should ignore any requests that are still running and could
    // reopen the panel once they resolve, because that would result in an
    // unsolicited UI behavior.
```

Reply:
Addressed in 8280fb6.